### ISSUE TITLE
Fatal error when connecting to Gitlab when access token has expired

### ISSUE THREAD

**Issue by markhowellsmead** [SayHelloGmbH/git-installer#69]
```
Fatal error: Uncaught Error: Cannot use object of type WP_Error as array in […]/wp-content/plugins/git-installer/src/Package/provider/Gitlab.php:149 
[…]
thrown in […]/wp-content/plugins/git-installer/src/Package/provider/Gitlab.php on line 149
```

**Comment by markhowellsmead** [comment]
Because this is a fatal error which blocks access to the whole backend, it's not possible to update the settings with a new token.

**Comment by nico-martin** [comment]
Should be solved with v1.3.2
